Dream-Home-PEABOT/FEE_DreamHome
# Om feature display generate report

### What does this PR do?
- Creates generate report component
- Styles generate report component

### Where should the reviewer start?
- `src/components/GenerateReport/GenerateReport.css`
- `src/components/GenerateReport/GenerateReport.tsx`

### How should this be manually tested?
- run `npm` start
- application should compile and the `GenerateReport` component should render on the DOM

### Any background context you want to provide?

### What are the relevant tickets?
closes #35 
### Questions:

### Screenshots (if appropriate)

<img width="1430" alt="Screen Shot 2021-01-07 at 10 24 30 AM" src="https://user-images.githubusercontent.com/56229864/103923602-8898c680-50d2-11eb-9484-5c35130f0619.png">
